modules/rosa-delete-cluster-admins.adoc

```diff
@@ -16,14 +16,14 @@ Only the user who created the cluster can revoke access for `cluster-admin` user
 
```

Comment:
There is no need to 'logged in to the cluster'. There is no steps bellow needed to 'logged in to the cluster'

Reply:
The 2nd. step in the procedure is to verify that the user no longer is listed in the output, so I'm retaining this piece of info.

Reply:
@sayjadha But the command in step2 'rosa list users --cluster=<cluster_name>' doesn't need login cluster. Do you want to check by 'oc get groups cluster-admins'? If so it needs login cluster.

Reply:
Thank you! Modified the command to `$ oc get groups cluster-admins` so we can #1: retain the login cluster requirement and #2: keep it consistent with the dedicated admin verification command.